lib/fizzy/execute.cpp

```diff
@@ -79,6 +79,21 @@ std::vector<uint64_t> execute(const module& _module, funcidx _function, std::vec
             stack.push(a + b);
             break;
         }
+        case instr::i32_wrap_i64: {
+            const auto a = static_cast<uint64_t>(stack.pop());
+            stack.push(a & 0x00000000ffffffff);
+            break;
+        }
+        case instr::i64_extend_i32_s: {
+            const auto a = static_cast<int64_t>(stack.pop());
+            stack.push(static_cast<uint64_t>(a));
+            break;
+        }
+        case instr::i64_extend_i32_u: {
```

Comment:
This should be effectively no-op, because we already zero-extend `i32` values when storing on stack. 
We can even drop this instruction in parser.

Reply:
We still need to meter it (at some point).